sirfilesystem.c

```diff
@@ -157,6 +157,86 @@ char* _sir_getcwd(void) {
 #endif
 }
 
+#if defined(__OpenBSD__)
```

Comment:
Just one last nitpick: can you move the implementation to the end of the file and put the declaration at the end of the header with a comment like the others?

Reply:
No problem.  Even though it's static?  Or you want it to be made non-static?  I even meant to declare it static inline but forgot.:)

Reply:
oh hmm, well, if you move it to the end of the source file w/o a declaration, getappfilename can't find it right? so i guess make it non-static and declare in the header?

Reply:
> oh hmm, well, if you move it to the end of the source file w/o a declaration, getappfilename can't find it right? so i guess make it non-static and declare in the header?

Yep, that works.